admin/class-primary-term-admin.php

```diff
@@ -105,7 +105,7 @@ protected function get_primary_term( $taxonomy_name ) {
 	protected function get_primary_term_taxonomies( $post_ID = null ) {
 
 		if ( null === $post_ID ) {
-			$post_ID = get_the_ID();
+			$post_ID = $_GET['post'];
```

Comment:
@Rarst yeah if we need it that'd be better, but still think using `get_queried_object_id` would be even better?